### PR TITLE
(GH-41) Added properties ThrowOnCancel, ThrowOnError and ThrowOnWarning

### LINKED ISSUE #41: Add common settings to SvnSettings
Add following properties to the SvnSettings (according SvnClientArgs) with corresponding default values.
ThrowOnCancel
ThrowOnError
ThrowOnWarning

### PR DESCRIPTION
**Pull request by x-jokay**
Fixes #41.

### CODE REVIEW

File: src/Cake.Svn/SvnSettings.cs [hunk 1]
```diff
@@ -5,5 +5,31 @@
...
+        /// Ctor.
+        /// </summary>
+        public SvnSettings()
+        {
+            ThrowOnCancel = true;
+            ThrowOnError = true;
+        }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if the
```
Comment (pascalberger) [inline src/Cake.Svn/SvnSettings.cs]: Should be `Gets or sets a value indicating...`

File: src/Cake.Svn/SvnSettings.cs [hunk 2]
```diff
@@ -5,5 +5,31 @@
...
+        }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if the 
+        /// operation is canceled. Defaults to true.
+        /// </summary>
+        public bool ThrowOnCancel { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if an
```
Comment (pascalberger) [inline src/Cake.Svn/SvnSettings.cs]: Should be `Gets or sets a value indicating...`

File: src/Cake.Svn/SvnSettings.cs [hunk 3]
```diff
@@ -5,5 +5,31 @@
...
+        /// error occurs. Defaults to true.
+        /// </summary>
+        public bool ThrowOnError { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if a non
```
Comment (pascalberger) [inline src/Cake.Svn/SvnSettings.cs]: Should be `Gets or sets a value indicating...`

File: src/Cake.Svn/SvnSettings.cs (hunk 1 above)
Comment (x-jokay) [reply src/Cake.Svn/SvnSettings.cs]: fixed.

File: src/Cake.Svn/SvnSettings.cs (hunk 2 above)
Comment (x-jokay) [reply src/Cake.Svn/SvnSettings.cs]: fixed.

File: src/Cake.Svn/SvnSettings.cs (hunk 3 above)
Comment (x-jokay) [reply src/Cake.Svn/SvnSettings.cs]: fixed.

**Pull request merged**